themes/morphosource/views/Browse/bibliography_list_html.php

```diff
@@ -32,9 +32,14 @@
 	<div id="browseList">
 		<div class="tealRule"><!-- empty --></div>
 		<H2><?php print _t("Bibliographic Citations"); ?></H2>
+		<div class='' style='padding-left: 10px;'>
+			Filter: <input type='text' name='filter' value='' onkeyup='filterBiblioBrowse(this.value); return false;' size='20' style='border:1px solid #828282; margin-right: 10px; margin-left: 5px;'/>
+			<span id='numberBiblioFiltered'></span>
+			<br/><br/>
+		</div><!-- end class list-filter -->
 <?php
 	if($q_bibliography->numRows() > 0){
-		print "<div id='browseListScrollContainer'>";
+		print "<div class='browseListScrollContainer' style='padding-right: 10px; border-top: 1px solid #578686; margin-top: 5px; padding-top: 20px;'>";
```

Comment:
just curious, I see that you changed id to class, but I don't see any new/update references to .browseListScrollContainer selector.  Just wondering the reason for the change and want to make sure it is expected.  Thanks.

Reply:
The project browse view already uses the browseListScrollContainer class selector, and the expected behavior is that this class makes the div scroll within the current window. I think whoever originally wrote the institution/biblio views just made a mistake and put id where they meant to put class, so that they would all handle that way.